mbtools/models.py

```diff
@@ -169,6 +171,25 @@ def __init__(self, parent, location):
         self.etree = html.fromstring(self.parent.text)
         # self.etree = html.fromstring(self.parent.text)
 
+    def replace_content_tag(self):
+        # ignore replaced elements
+        attrib_dict = self.parent.attrib
+        if self.parent.tag in ["content", "contents"]:
+            if "class" in attrib_dict.keys() and \
+                    attrib_dict["class"] == "os-raise-content":
```

Comment:
:thinking: - Is this looking at the right set of attributes since I think `attrib_dict` comes from the parent `<content>` / `<contents>`? I believe what you really want is to check the attributes on the element in `self.etree`.

Perhaps my reading of the code is wrong, but I don't think there's a test case which hits L180 so either it's not working or there's not a specific test to check for this code path :man_shrugging: .

Reply:
Good catch! I fixed this and added a test.